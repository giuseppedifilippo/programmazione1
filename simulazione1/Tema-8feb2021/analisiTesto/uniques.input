Vivam laoreets, ac elementum. Vivams ullamcorper ligulas ex, velt malesuada odio rutrum eu. Nunct veml mollis. Phasellus ut laoreet lectus. Nulla conguet nuncu vitae justo sagittis, quis dapibum dolors tristique. Mauriset ut lobortis magna. Pellentesque congue maliquam sem bibendum.  Fusce ut risus ligula fringilla tempor. Aliquamt et orcis urnam. In sit amets consectetur mi. Integer dapibus mauris consequat. Vestibulum ante ipsums primis in faucibus orci luctus et ultrices posueret cubilia curae; Nunc ipsum urnas, posuere sits amet eleifend ut, efficitur at dui. Aliquam tincidunt leo a dolor ultricies, sed maximus metus placerat.

ReadFile files named by filename and return contents. A successful call returns, not err endof. Because it reads the whole file, it doesn't treat an EOF from Read as an error to be reported.
