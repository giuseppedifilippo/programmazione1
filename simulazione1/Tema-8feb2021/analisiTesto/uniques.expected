A : 1
Aliquam : 1
Aliquamt : 1
Because : 1
EOF : 1
Fusce : 1
In : 1
Integer : 1
Mauriset : 1
Nulla : 1
Nunc : 1
Nunct : 1
Pellentesque : 1
Phasellus : 1
Read : 1
ReadFile : 1
Vestibulum : 1
Vivam : 1
Vivams : 1
a : 1
ac : 1
amet : 1
amets : 1
an : 2
and : 1
ante : 1
as : 1
at : 1
be : 1
bibendum. : 1
by : 1
call : 1
congue : 1
conguet : 1
consectetur : 1
consequat. : 1
contents. : 1
cubilia : 1
curae; : 1
dapibum : 1
dapibus : 1
doesn't : 1
dolor : 1
dolors : 1
dui. : 1
efficitur : 1
eleifend : 1
elementum. : 1
endof. : 1
err : 1
error : 1
et : 2
eu. : 1
ex, : 1
faucibus : 1
file, : 1
filename : 1
files : 1
fringilla : 1
from : 1
in : 1
ipsum : 1
ipsums : 1
it : 2
justo : 1
laoreet : 1
laoreets, : 1
lectus. : 1
leo : 1
ligula : 1
ligulas : 1
lobortis : 1
luctus : 1
magna. : 1
malesuada : 1
maliquam : 1
mauris : 1
maximus : 1
metus : 1
mi. : 1
mollis. : 1
named : 1
not : 1
nuncu : 1
odio : 1
orci : 1
orcis : 1
placerat. : 1
posuere : 1
posueret : 1
primis : 1
quis : 1
reads : 1
reported. : 1
return : 1
returns, : 1
risus : 1
rutrum : 1
sagittis, : 1
sed : 1
sem : 1
sit : 1
sits : 1
successful : 1
tempor. : 1
the : 1
tincidunt : 1
to : 1
treat : 1
tristique. : 1
ullamcorper : 1
ultrices : 1
ultricies, : 1
urnam. : 1
urnas, : 1
ut : 3
ut, : 1
velt : 1
veml : 1
vitae : 1
whole : 1
1 3 1.0413223140495869
[Read ReadFile]
